app/services/ContentClient.scala

```diff
@@ -0,0 +1,42 @@
+package services
+
+import com.gu.contentapi.client.model.v1.{SearchResponse, SectionsResponse, TagsResponse}
+import com.gu.contentapi.client.{ContentApiClient, GuardianContentClient}
+import play.api.libs.json.{Json, Reads, Writes}
+
+import scala.concurrent.{ExecutionContext, Future}
+
+object CapiSearchParams {
+  implicit val writes: Writes[CapiSearchParams] = Json.writes[CapiSearchParams]
+  implicit val reads: Reads[CapiSearchParams] = Json.reads[CapiSearchParams]
+}
+
+case class CapiSearchParams(query: String, tags: Option[List[String]] = None, sections: Option[List[String]] = None)
```

Comment:
I think the types can be simpler here. `Option[List[String]]` could be a `List[String]` with a default value of `List.empty`.

Reply:
True. This is the result of me servicify-ing controller code – IIRC using `List.empty` results in a non-optional field when deserialising from JSON, so we'll need a new type in the controller, but that's fine 👍

Reply:
Yeah, making the controller a translator from request and all it's properties into strictly typed cases classes makes this easier to reason about and test IMO.